ReadMe.md

```diff
@@ -262,13 +274,16 @@ Capacitor | コンデンサ | | Alternatively: キャパシタ
 Circuit | 回路 | かいろ |
 Circuit diagram (Schematic) | 回路図 | かいろず |
 Component placement diagram | 部品配置図 | ぶひんはいちず
+Communications | 通信 | つうしん |
 Conduction | 導電 | どうでん |
 Conductivity | 導電性 | どうでんせい |
 Connector | コネクタ | |
 Current | 電流 | でんりゅう |
 Datasheet | データシート | |
 Diode | ダイオード | |
 Drawing | 図面 | ずめん |
+Electrolytic	| 電解 | だんかい |
+Frequency	| 周波数 | しゅうはすう|
```

Comment:
Sorry to be pedantic, but could you add a space before the closing `|`, and remove the extra whitespace in the first column?